config/common/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml

```
@@ -751,3 +751,10 @@ local_volume_provisioner_storage_classes: # Installation via kubespray has no su
 
 local_volume_provisioner_nodelabels: []
 local_volume_provisioner_tolerations: []
+
+ntp_filter_interface: false
```

Shouldn't this be true?
```suggestion
ntp_filter_interface: true
```

---

Yes, True. I wanted the operator to flip the switch, so they don't accidentally leave the ntp_interfaces as it is and then if ens3 does not exist, NTP would fail syncing the time. I added a readme line that asks the operator to set this to true..

---

Now that I think about it, I remember doing a quick check across all providers we have and the default interface is ens3. I will set this to true by default then. :)

---

I don't think that we can assume that every user of this repo uses ens3 as the default interface. What do you think about adding a note in the migration guide to disable it if they aren't using ens3 as the default interface?

---

Okay, sure that sounds like a good approach!